docs/README.md

```diff
@@ -2,6 +2,9 @@
 
 ## Overview
 
-This folder contains documents that relate to the project.
+The docs folder contains two subfolders - `user` and `contributor`. 
+The `user` subfolder contains the end-user documentation, including any overview, usage or troubleshooting documents. The content of this folder is displayed on the [Kyma website](https://kyma-project.io/#/)
 
-> In this document, provide the list of all documents that this folder contains, together with links to those documents and short information on what they describe.
+The `contributor` subfolder includes any developer-related documentation that can help in improving the given module's usage.
```

Comment:
`that can help in improving the given module's usage.` sounds strange for me, is that needed?

We have the scenarios of contributing==developing the module, operating the module and manual installation of the module, that should be outlined more transparent